x/fees/types/params.go

```diff
@@ -0,0 +1,108 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramsModule "github.com/cosmos/cosmos-sdk/x/params/subspace"
+)
+
+const (
+	// default paramspace for paramsModule keeper
+	DefaultParamspace = ModuleName
+)
+
+var (
+	DefaultFeeDenom = sdk.DefaultBondDenom
+	DefaultMinFees  []MinFee
+)
+
+// Parameters store keys
+var (
+	FeeDenomStoreKey = []byte("FeeDenom")
+	MinFeesStoreKey  = []byte("MinFees")
+)
+
+// ParamKeyTable Key declaration for parameters
+func ParamKeyTable() paramsModule.KeyTable {
+	return paramsModule.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+type Params struct {
+	FeeDenom string   `json:"fee_denom" yaml:"fee_denom"`
```

Comment:
I think that if the `MinFees.Amount` value is changed to use `sdk.Coins`, this can be safely deleted